test/8.js

```diff
@@ -0,0 +1,19 @@
+const solution = require('../solutions/8.js');
+
+const testPrime = (num, result)=>{
+  if(solution(num) === result){
```

Comment:
You can just do `solution(num)`. `Solution` should return true or false anyways.